include/swift/AST/DiagnosticsSema.def

```diff
@@ -3296,6 +3296,35 @@ NOTE(availability_conformance_introduced_here, none,
      "conformance introduced here", ())
 
 //------------------------------------------------------------------------------
+// Resilience diagnostics
+//------------------------------------------------------------------------------
+
+ERROR(versioned_attr_with_explicit_accessibility,
+      none, "'@_versioned' attribute can only be applied to internal "
+      "declarations, but %0 is %select{private|fileprivate|internal|public}1",
+      (Identifier, unsigned))
+
+#define FRAGILE_FUNC_KIND \
+  "%select{a '@_transparent' function|" \
+  "an '@inline(__always)' function|" \
+  "an '@_inlineable' function|" \
+  "a default argument value}"
+
+ERROR(local_type_in_inlineable_function,
+      none, "type %0 cannot be nested inside " FRAGILE_FUNC_KIND "1",
+      (DeclName, unsigned))
+
+ERROR(resilience_decl_unavailable,
+      none, "%0 %1 is %select{internal|private}2 and cannot be referenced "
```

Comment:
This leaves out `fileprivate`. Can you use Accessibility as the argument here, even if `public` and `open` don't make sense?